src/Stateless/ShapeColor.cs

```diff
@@ -0,0 +1,137 @@
+﻿namespace Stateless.Dot
+{
+    enum ShapeColor
```

Comment:
Since these are only used directly in a few places, e.g. literal `ShapeColor.blue`, would it make sense to use a couple of string constants instead?